exercises/kindergarten-garden/src/main/java/KindergartenGarden.java

```diff
@@ -0,0 +1,7 @@
+class KindergartenGarden {
+
+    List<Plant> getPlantsOfStudent(String student) {
```

Comment:
You're missing an import for `List` which is why the build failed :) You also need to add the two constructors used in the tests :)

Reply:
Ah, wasn't sure if we wanted the constructors or not! I'll get on it now :slightly_smiling_face:

Reply:
We want to add enough to the starter implementation so that the tests will compile without errors. If we didn't add constructors then the tests would have errors :) We want users to be able to run the tests straight away without having to deal with scary errors first :)

Reply:
Makes sense to me :smile: